truss/cli/cli.py

```diff
@@ -533,20 +574,103 @@ def deploy(
                 elif num_failed := num_services - num_active - num_deploying:
                     break
                 time.sleep(status_check_wait_sec)
-        # Print must be outside `Live` context.
+
+            intercepted_logs = log_interceptor.get_logs()
+
+        # Prints must be outside `Live` context.
+        if intercepted_logs:
```

Comment:
ok nice -- this solves the problem where logs were screwing up the table view, right?